### PR TITLE
Fix/tests ci

### PR DESCRIPTION
**Pull request by ogarciarevett**
This PR is co-related with this one:
https://github.com/lootledger/inventory/pull/15

Fixing tests here and adding CI steps to maintain coverage and tests

### CODE REVIEW

File: contracts/libraries/LibInventory.sol [hunk 1]
```diff
@@ -6,16 +6,15 @@ LibInventory defines the storage structure used by the Inventory contract as a f
 proxy.
  */
 library LibInventory {
-  bytes32 constant STORAGE_POSITION = keccak256("summon.eth.storage.Inventory");
+  bytes32 constant STORAGE_POSITION = keccak256("eth.storage.Inventory");
```
Comment (zcstarr) [inline contracts/libraries/LibInventory.sol:9]: nit: minor but maybe we would call this lootledger.storage.Inventory vs eth as a prefix, just incase anyone is already using that position.

File: contracts/Inventory.sol [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+    bytes32 public constant INVENTORY_ADMIN = keccak256("INVENTORY_ADMIN");
+
+    address public contractERC721Address;
+    uint256 public numSlots;
+
+    // SlotId => slot, useful to get the rest of the slot data.
+    mapping(uint256 => LibInventory.Slot) private slotData;
+
+    // SlotType => "slot type name"
+    mapping(uint256 => string) private slotTypes;
```
Comment (zcstarr) [inline contracts/Inventory.sol:37]: Is slot type deprecated? I noticed it's not there within any of the InventoryFacets

**Review by zcstarr — COMMENTED** [review]
Dropped a few questions, in the PR. and then I have this final one 

Is now a good time to bump to ethers v6, alot of changes but good ones going forward, we could always land this first then bump. 

It'll change the deploy script quite a bit as the interface for interacting with contracts changes between the two versions. It might also have an affect downstream. 

If you want to land this then update I'm down for that. Overall lgtm except those 2 minor comments 👍

File: contracts/libraries/LibInventory.sol (hunk 1 above)
Comment (ogarciarevett) [reply contracts/libraries/LibInventory.sol:9]: yeah well, probably will be something different that "lootledger" (spoiler alert)

File: contracts/Inventory.sol (hunk 2 above)
Comment (ogarciarevett) [reply contracts/Inventory.sol:37]: Yes, the slot type will be an "extension" actually this shouldn't be here, thanks for the comment.

**Pull request merged**